hail/python/hail/matrixtable.py

```diff
@@ -2500,7 +2500,7 @@ def __init__(self, table, n_rows, n_cols, width, truncate, types):
 
         def __str__(self):
             s = self.table_show.__str__()
-            s += f"showing first { self.n_cols } { plural('column', self.n_cols) }"
+            s += f"showing the first { self.n_cols } columns"
```

Comment:
you shouldn't print this message at all if the number of MT cols is smaller than n_cols, I think